dev/SourceProcessors.md

```diff
@@ -0,0 +1,153 @@
+# Source Processors
+
+This document explains how _source processors_ work and how to configure a simple local demo using
+a _SlackSource_ and a _SlackAction_ (called _"From Slack To Slack"_).
+
+## How to define a source processor
+
+The payload to define a source processor is similar to a sink processor, with two differences:
+
+* It contains a _"source"_ field instead of _"action"_. The internal structure is the same, with _"type"_ and _"parameters"_.
+* It currently ignores any specified _"transformationTemplate"_.
+
+Filters work exactly in the same way.
+
+### Example payload to create a source processor
+
+```json
+{
+  "name": "slack-source",
+  "source": {
+    "type": "Slack",
+    "parameters": {
+      "channel": "my-read-channel",
+      "token": "xoxb-..."
+    }
+  },
+  "filters": [
+    {
+      "key": "data.text",
+      "type": "StringContains",
+      "values": [ "hello", "hi" ]
+    }
+  ]
+}
+```
+
+## Currently implemented sources
```

Comment:
@kostola We have `ACTIONS.md` in the root of the repo for the list of `Actions` we support. I think we should have something similar for the list of `Sources` that we support. Our other general user-facing docs e.g `ACTIONS.md`, `FILTERS.md` and `TRANSFORMATIONS.md` will also need updating to reflect the introduction of `Source Processors`. 

Those `.md` files are useful for the docs team who are doing their initial first iteration of customer-facing content now.

cheers,

Rob

Reply:
I updated all these files and added a new `SOURCES.md` file :+1: